Student/StudentE2ETests/Login/LoginE2ETests.swift

```diff
@@ -66,6 +65,10 @@ class LoginE2ETests: CoreUITestCase {
         emailField.typeText("\(user.username)\r")
         LoginWeb.passwordField.tap()
         LoginWeb.passwordField.typeText("\(user.password)\r")
+        let staySignedInButton = app.find(label: "No", type: .button)
```

Comment:
Microsoft added a "stay logged in" screen between the login and the app. The "remember my decision" option seemed to do nothing.